src/main/java/com/minecolonies/coremod/colony/buildings/workerbuildings/BuildingLumberjack.java

```diff
@@ -212,17 +212,18 @@ public AbstractJob createJob(final CitizenData citizen)
     public void readFromNBT(@NotNull final NBTTagCompound compound)
     {
         super.readFromNBT(compound);
-        if (treesToFell.isEmpty())
+        final NBTTagList saplingTagList = compound.getTagList(TAG_SAPLINGS, Constants.NBT.TAG_COMPOUND);
+        for (int i = 0; i < saplingTagList.tagCount(); ++i)
         {
-            final NBTTagList saplingTagList = compound.getTagList(TAG_SAPLINGS, Constants.NBT.TAG_COMPOUND);
-            for (int i = 0; i < saplingTagList.tagCount(); ++i)
-            {
-                final NBTTagCompound saplingCompound = saplingTagList.getCompoundTagAt(i);
-                final ItemStack stack = new ItemStack(saplingCompound);
-                final boolean cut = saplingCompound.getBoolean(TAG_CUT);
-                treesToFell.put(new ItemStorage(stack), cut);
-            }
+            final NBTTagCompound saplingCompound = saplingTagList.getCompoundTagAt(i);
+            final ItemStack stack = new ItemStack(saplingCompound);
+            final boolean cut = saplingCompound.getBoolean(TAG_CUT);
+            final ItemStorage storage = new ItemStorage(stack); 
+            if (treesToFell.containsKey(storage)) {
```

Comment:
formatting to 
{

}
 
please to blend in with the rest

Reply:
updated formatting.